src/containers/ArticlePage/LearningResourcePage/components/LearningResourceTaxonomy.jsx

```diff
@@ -452,7 +452,7 @@ class LearningResourceTaxonomy extends Component {
           <SaveButton
             isSaving={status === 'loading'}
             showSaved={status === 'success' && !isDirty}
-            disabled={!isDirty || !resourceTypes.length}
+            disabled={!isDirty}
```

Comment:
Må nesten legge tilbake denne.

Reply:
Men du kan fjerne valget for emner. Det er ikkje lenger behov å kunne sette ressurstype på emne.

Reply:
ah okei, skjønner da legger jeg den tilbake! De som bruker det vet kanskje at de må velge ressurstype... Men mente du å fjerne hele `ResourceTypeSelect` fra topicarticle?

Reply:
Ja, for det var en ting som blei lagt til som en midlertidig løsning fordi ndla-film brukte emneartikler i en tidligere versjon. Så vi trengte det for å søke dei opp på film-sida. No er dette endra til å bruke ressurser, så då trengs det ikkje lenger. Kan alltids ta det i en egen pr.

Reply:
Okei, lager en egen pr på det, for den brukte en del props, funksjoner etc. som er greit å fjerne i samme slengen!